report-jtreg-comparator/src/main/java/io/jenkins/plugins/report/jtreg/main/comparator/VariantComparator.java

```diff
@@ -6,13 +6,20 @@
 
 import java.io.File;
 import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
 import java.util.List;
 
 public class VariantComparator {
+
+    static List<String> copyOfArgs;
+
     public static void main(String[] args) throws Exception {
+        copyOfArgs = Collections.unmodifiableList(Arrays.asList(args));
```

Comment:
Please add comment explaining why iot is here. eg "IF we do not make a copy of original arguments, we would need to reconstruct the it back from (generic)options and from ComapratorOption, which would be a lot of work"